apps/site/pages/en/learn/asynchronous-work/discover-javascript-timers.md

```diff
@@ -87,6 +93,8 @@ const id = setInterval(() => {
 clearInterval(id);
 ```
 
+Like `setTimeout`, `setInterval` returns a Timeout object in Node.js (and a numeric ID in browsers). This object provides the same additional methods (`ref()`, `unref()`, and `refresh()`).
+
```

Comment:
```suggestion
```

Earlier, we say "`setInterval` is a function similar to `setTimeout`", which I think implies they have the same return.